tests/actions/IOUTest.js

```diff
@@ -683,7 +683,7 @@ describe('actions/IOU', () => {
                                         Onyx.disconnect(connectionID);
                                         expect(transaction.pendingAction).toBeFalsy();
                                         expect(transaction.errors).toBeTruthy();
-                                        expect(_.values(transaction.errors)[0]).toBe('iou.error.genericCreateFailureMessage');
+                                        expect(_.values(transaction.errors)[0]).toStrictEqual(['iou.error.genericCreateFailureMessage', {isTranslated: false}]);
```

Comment:
Any reason for using `toStrictEqual`?
I see `toEqual` in other parts (i.e. [here](https://github.com/Expensify/App/pull/30073/files#diff-a7f0b97fed238bc21653cb5b40f84a094a4fcc2d3ffcab4317bb8ca0db97e35eR1631) and [here](https://github.com/Expensify/App/pull/30073/files#diff-a7f0b97fed238bc21653cb5b40f84a094a4fcc2d3ffcab4317bb8ca0db97e35eR1845))